bitcoin/src/blockdata/locktime/absolute.rs

```diff
@@ -146,6 +152,13 @@ impl LockTime {
 
     /// Constructs a `LockTime` from `n`, expecting `n` to be a valid block time.
     ///
+    /// ## Note on Locktime and Block Inclusion:
+    ///
+    /// If the current median time of the past eleven blocks is `T` and the locktime is set to `T`,
+    /// the transaction can be included in the following block.
```

Comment:
I don't think this is exactly right -- once the median time of the last 11 blocks is `T`, then the timestamp of the next block is allowed to be `T` (but probably it will actually be greater than `T`, assuming the miner is trying to target the correct time). But the transaction won't be included unless the block timestamp actually *is* `T`.

But I'm unsure.

Reply:
See https://github.com/rust-bitcoin/rust-bitcoin/pull/2858#issuecomment-2162216597
https://github.com/rust-bitcoin/rust-bitcoin/pull/2858#issuecomment-2162236275